deployments/base/usdc/migrations/1689892563_configurate_and_ens.ts

```diff
@@ -0,0 +1,242 @@
+
+    const setConfigurationCalldata = await calldata(
+      configurator.populateTransaction.setConfiguration(comet.address, configuration)
+    );
+    const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [configurator.address, comet.address]
+    );
+    const setRewardConfigCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address'],
+      [comet.address, baseCOMPAddress]
+    );
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [configurator.address, cometAdmin.address, rewards.address],
+        [0, 0, 0],
+        [
+          'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
+          'deployAndUpgradeTo(address,address)',
+          'setRewardConfig(address,address)'
+        ],
+        [setConfigurationCalldata, deployAndUpgradeToCalldata, setRewardConfigCalldata]
+      ]
+    );
+
+    const COMPAmountToBridge = exp(12_500, 18);
+    const USDCAmountToBridge = exp(10_000, 6);
+
+    const actions = [
+      // 1. Set Comet configuration + deployAndUpgradeTo new Comet and set reward config on Base.
+      {
+        contract: baseL1CrossDomainMessenger,
+        signature: 'sendMessage(address,bytes,uint32)',
+        args: [bridgeReceiver.address, l2ProposalData, 2_500_000]
+      },
+
+      // 2. Approve Ethereum's L1StandardBridge to take Timelock's USDC (for bridging)
+      {
+        contract: mainnetUSDC,
+        signature: 'approve(address,uint256)',
+        args: [baseL1StandardBridge.address, USDCAmountToBridge]
+      },
+      // 3. Bridge USDC from Ethereum to Base Comet using L1StandardBridge
+      {
+        contract: baseL1StandardBridge,
+        // function depositERC20To(address _l1Token, address _l2Token, address _to, uint256 _amount, uint32 _l2Gas,bytes calldata _data)
+        signature: 'depositERC20To(address,address,address,uint256,uint32,bytes)',
+        args: [mainnetUSDC.address, USDC.address, comet.address, USDCAmountToBridge, 200_000, '0x']
+      },
+
+      // 4. Approve Ethereum's L1StandardBridge to take Timelock's COMP (for bridging)
+      {
+        contract: mainnetCOMP,
+        signature: 'approve(address,uint256)',
+        args: [baseL1StandardBridge.address, COMPAmountToBridge]
+      },
+      // 5. Bridge COMP from Ethereum to Base Rewards using L1StandardBridge
+      {
+        contract: baseL1StandardBridge,
+        // function depositERC20To(address _l1Token, address _l2Token, address _to, uint256 _amount, uint32 _l2Gas,bytes calldata _data)
+        signature: 'depositERC20To(address,address,address,uint256,uint32,bytes)',
+        args: [mainnetCOMP.address, baseCOMPAddress, rewards.address, COMPAmountToBridge, 200_000, '0x']
+      },
```

Comment:
These are not needed. Gauntlet has been replenishing the rewards contract on Base.